input/fsh/extensions/HomeMunicipalityExtension.fsh

```diff
@@ -0,0 +1,18 @@
+Extension: HomeMunicipalityExtension
+Id: HomeMunicipalityExtension
+Description: "An extension for patient home municipality."
```

Comment:
This should actually be defined in the base profiles, and we should refer to that extension.